frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/manyOfProperty.component.tsx

```diff
@@ -17,18 +17,17 @@
 
 import { TeamspacesHooksSelectors } from '@/v5/services/selectorsHooks/teamspacesSelectors.hooks';
 import { UsersHooksSelectors } from '@/v5/services/selectorsHooks/usersSelectors.hooks';
-import { PropertyDefinition } from '@/v5/store/tickets/tickets.types';
 import { FormMultiSelect } from '@controls/formMultiSelect/formMultiSelect.component';
 import { MultiSelectMenuItem } from '@controls/formMultiSelect/multiSelectMenuItem/multiSelectMenuItem.component';
 import { store } from '@/v4/modules/store';
 import _ from 'lodash';
+import { PropertyProps } from './properties.types';
 
 export const ManyOfProperty = ({
 	property: { name, readOnly, required, values },
-	value: defaultValues,
-}: { property: PropertyDefinition, value: any }) => {
-	const selectedValues = (defaultValues || []) as string[];
-
+	defaultValue = [],
+	...props
+}: PropertyProps) => {
 	if (values === 'jobsAndUsers') {
```

Comment:
regarding this. I would create a basic Many off (which would use values as values. And a JobsAndUsers select. This component would return one or the other